libraries/AP_HAL_ChibiOS/hwdef/MambaF405US-I2C/hwdef.dat

```diff
@@ -0,0 +1,167 @@
+PB6 USART1_TX USART1 NODMA ALT(1)
+
+# SBUS inversion control pin, active low
+PC0 USART1_RXINV OUTPUT HIGH GPIO(78) POL(0)
+
+# USART2
+PA2 USART2_TX USART2
+PA3 USART2_RX USART2
+
+# USART3
+PB10 USART3_TX USART3
+PB11 USART3_RX USART3
+
+# UART4
+PA0 UART4_TX UART4
+PA1 UART4_RX UART4
+
+# UART5
+# These are labeled in betaflight F405US_I2C target but aren't on the board?
+# I'm assuming there are more boards that use this target
+PC12 UART5_TX UART5
+PD2 UART5_RX UART5
+
+# USART6
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6
+
+# ADC
+PC1 BAT_VOLT_SENS ADC1 SCALE(1)
+PC2 RSSI_IN ADC1
+PC3 BAT_CURR_SENS ADC1 SCALE(1)
+
+# PWM output. 1 - 4 on ESC header
+PA9 TIM1_CH2 TIM1 PWM(1) GPIO(50)
+PA8 TIM1_CH1 TIM1 PWM(2) GPIO(51)
+PC9 TIM3_CH4 TIM3 PWM(3) GPIO(52)
+PC8 TIM3_CH3 TIM3 PWM(4) GPIO(53)
+
+# Board LEDs
+PC15 LED1 OUTPUT LOW GPIO(1)
+PC14 LED2 OUTPUT LOW GPIO(2)
+define HAL_GPIO_A_LED_PIN 1
+define HAL_GPIO_B_LED_PIN 2
+
+# External LEDs
+PB3 LED_EXT1 OUTPUT GPIO(30)
+
+# Buzzer
+PC13 BUZZER OUTPUT GPIO(80) LOW
+define HAL_BUZZER_PIN 80
+define HAL_BUZZER_ON 1
+define HAL_BUZZER_OFF 0
+
+# Note that this board needs PULLUP on I2C pins
+PB8 I2C1_SCL I2C1 PULLUP
+PB9 I2C1_SDA I2C1 PULLUP
+
+# SPI1 - Internal IMU
+PA4 MPU6000_CS CS
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+# SPI2 - OSD
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+# OSD max7456
+PB12 OSD_CS CS
+
+# SPI3 - dataflash
+PC10 SPI3_SCK SPI3
+PC11 SPI3_MISO SPI3
+PB5 SPI3_MOSI SPI3
+# Dataflash M25P16
+PA15 FLASH_CS CS
+
+# SPI Device table
+SPIDEV mpu6000    SPI1 DEVID1 MPU6000_CS MODE3 1*MHZ 4*MHZ
+SPIDEV osd        SPI2 DEVID4 OSD_CS MODE0 10*MHZ 10*MHZ
+SPIDEV dataflash  SPI3 DEVID1 FLASH_CS   MODE3 32*MHZ 32*MHZ
+
+# One IMU rotated in yaw
+IMU Invensense SPI:mpu6000 ROTATION_YAW_90
+
+# Probe for I2C BMP280, but allow init on board variants without onboard baro too
+BARO BMP280 I2C:0:0x76
+define HAL_PROBE_EXTERNAL_I2C_BAROS
+define HAL_BARO_ALLOW_INIT_NO_BARO
+
+# no built-in compass, but probe the i2c bus for all possible
+# external compass types
+define ALLOW_ARM_NO_COMPASS
+define HAL_COMPASS_DEFAULT HAL_COMPASS_NONE
+define HAL_PROBE_EXTERNAL_I2C_COMPASSES
+define HAL_I2C_INTERNAL_MASK 0
+define HAL_COMPASS_AUTO_ROT_DEFAULT 2
+
+# enable logging to dataflash
+define HAL_LOGGING_DATAFLASH
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+# flash size
+FLASH_SIZE_KB 1024
+# reserve 32k for bootloader and 32k for flash storage
+FLASH_RESERVE_START_KB 64
+
+# define default battery setup
+define HAL_BATT_VOLT_PIN 11
+define HAL_BATT_CURR_PIN 13
+define HAL_BATT_VOLT_SCALE 11
+define HAL_BATT_CURR_SCALE 39
+
+# Analog RSSI pin (also could be used as analog airspeed input)
+define BOARD_RSSI_ANA_PIN 1
+
+# Setup for OSD
+define OSD_ENABLED 1
+define HAL_OSD_TYPE_DEFAULT 1
+# Font for the osd
+ROMFS_WILDCARD libraries/AP_OSD/fonts/font*.bin
+
+# To complementary channels work we define this
+#define STM32_PWM_USE_ADVANCED TRUE
```

Comment:
this should be automatic now

Reply:
These both should be good to go now

Reply:
I'm not sure whether you were talking about the romfs wildcard or both, I removed both but can revert